Xdl.Internship.Core/Xdl.Internship.Core.DataAccess.MongoDB/Settings/IMongoDBSetting.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Xdl.Internship.Core.DataAccess.MongoDB
+{
+    public interface IMongoDBSetting
```

Comment:
class should have properties only for connection parameters for MongoClient, but not names of options from App config

Reply:
We can keep class only, interface here has no real reason

Reply:
pls remove interface, keep only class